arrows/pytorch/grid.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+import torch
+
+
+class grid(object):
+    def __init__(self, grid_row=15, grid_cols=15, target_neighborhood_w=7):
+        self._grid_rows = grid_row
+        self._grid_cols = grid_cols
+        self._target_neighborhood_w = target_neighborhood_w
+        self._half_cell_w = int(self._target_neighborhood_w // 2)
+
+    def __call__(self, im_size ,bbox_list, MOT_flag=False):
+        return self.obtainGridFeatureList(im_size, bbox_list, MOT_flag)
+
+    @property
+    def img_h(self):
+        return self._img_h
+
+    @img_h.setter
+    def img_h(self, val):
+        self._img_h = val
+
+    @property
+    def img_w(self):
+        return self._img_w
+
+    @img_w.setter
+    def img_w(self, val):
+        self._img_w = val
```

Comment:
These properties are trivial, so they can be removed and code in this class can use `img_h` and `img_w` directly.